2023-10-17/main.go

```diff
@@ -0,0 +1,22 @@
+package main
+
+import "fmt"
+
+func concatenate(a, b string) string {
+	return a + b
+}
+
+func swap(a, b string) (x, y string) {
```

Comment:
`func swap(a, b string)`
the parameters a, b, variables containing strings

`(x, y string) {`
in the return type we define that we want a return with variables x and y as strings